types/scrivito/lib/Attribute.d.ts

```diff
@@ -0,0 +1,23 @@
+export type Attribute =
```

Comment:
These `lib` files don't actually exist in the package. Can you inline them please?

Reply:
yes, I'll see to it today. What do you think about a type only export (provided we use Typescript 3.8 and newer)

Reply:
@DanielRosenwasser I checked, the typings you see are exported only in Attribute.d.ts to be used in other definition files. They are not exported to the public facing API, e.g you can not do

```
import { Attribute } from 'scrivito'
```